core/tests/protractor_utils/ExplorationEditorHistoryTab.js

```diff
@@ -86,7 +86,8 @@ var ExplorationEditorHistoryTab = function() {
         var matched = false;
         for (var i = 0; i < listOfNames.length; i++) {
           if (listOfNames[i] === stateName) {
-            await stateNodes.get(i).click();
+            var getStateNodes = stateNodes.get(i);
+            await action.click('Get State Nodes', getStateNodes);
```

Comment:
The first argument represents what the element is called. See here:
https://github.com/oppia/oppia/blob/668193808fd53189f2643013676cec63f9a25747/core/tests/protractor_utils/action.js#L26-L33

So perhaps a better name would be "State node".

Reply:
Ok I will change it.